manifests/pool.pp

```diff
@@ -14,6 +14,7 @@
   Optional[Integer] $mtu                    = undef,
   String $domain_name                       = '',
   $ignore_unknown                           = undef,
+  Optional[Integer] $max_lease_time         = undef,
```

Comment:
Adding this parameter isn't related to changing the above datatype, right? In that case, can you please submit it as a new pull request?